app/main.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import sys
+from structlog import wrap_logger
+
+from sdc.rabbit import MessageConsumer
+from sdc.rabbit import QueuePublisher
+
+from app.response_processor import ResponseProcessor
+import app.settings
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def run():
+    logging.getLogger("pika").setLevel(logging.INFO)
```

Comment:
Need to remove setting log level for pika logger, causes problems with picking up logs in splunk.